<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MapEditor</class>
 <widget class="QMainWindow" name="MapEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QTableWidget" name="tableWidget">
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>40</number>
        </attribute>
        <attribute name="horizontalHeaderMinimumSectionSize">
         <number>40</number>
        </attribute>
        <attribute name="verticalHeaderDefaultSectionSize">
         <number>40</number>
        </attribute>
        <attribute name="verticalHeaderMinimumSectionSize">
         <number>40</number>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QProgressBar" name="progressBar">
        <property name="value">
         <number>100</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_Open"/>
    <addaction name="action_Save"/>
    <addaction name="separator"/>
    <addaction name="action_Quit"/>
   </widget>
   <widget class="QMenu" name="menu_Tool">
    <property name="title">
     <string>&amp;Tool</string>
    </property>
    <addaction name="actionTool_Dialog"/>
    <addaction name="separator"/>
    <addaction name="actionAdd_new_row"/>
    <addaction name="actionAdd_new_Column"/>
    <addaction name="separator"/>
    <addaction name="actionSize"/>
    <addaction name="separator"/>
    <addaction name="action_Bat"/>
    <addaction name="separator"/>
    <addaction name="actionGo_to_cell"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="action_About_us"/>
    <addaction name="actionVersion"/>
   </widget>
   <widget class="QMenu" name="menu_Shadow">
    <property name="title">
     <string>&amp;Shadow</string>
    </property>
    <addaction name="actionTop"/>
    <addaction name="actionButton"/>
    <addaction name="actionLeft"/>
    <addaction name="actionRight"/>
    <addaction name="separator"/>
    <addaction name="actionTop_left"/>
    <addaction name="actionTop_right"/>
    <addaction name="actionButton_left"/>
    <addaction name="actionButton_right"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Tool"/>
   <addaction name="menu_Shadow"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QDockWidget" name="dockWidget">
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="nullButton">
       <property name="text">
        <string>Null</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="grass">
       <property name="text">
        <string>grass</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="VGlass">
       <property name="text">
        <string>VGlass</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="HGlass">
       <property name="text">
        <string>HGlass</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="VWall">
       <property name="text">
        <string>VWall</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="HWall">
       <property name="text">
        <string>HWall</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="VDoor">
       <property name="text">
        <string>VDoor</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="HDoor">
       <property name="text">
        <string>HDoor</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Floor">
       <property name="text">
        <string>Floor</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionNew">
   <property name="text">
    <string>&amp;New</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>&amp;Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>&amp;Save</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>quit</string>
   </property>
  </action>
  <action name="actionQuit_2">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </action>
  <action name="actionFloor">
   <property name="text">
    <string>floor</string>
   </property>
  </action>
  <action name="actionDoor">
   <property name="text">
    <string>door</string>
   </property>
  </action>
  <action name="actionWindow">
   <property name="text">
    <string>window</string>
   </property>
  </action>
  <action name="actionTree">
   <property name="text">
    <string>tree</string>
   </property>
  </action>
  <action name="actionFlower">
   <property name="text">
    <string>flower</string>
   </property>
  </action>
  <action name="actionGrass">
   <property name="text">
    <string>grass</string>
   </property>
  </action>
  <action name="action_Tool">
   <property name="text">
    <string>&amp;Tool</string>
   </property>
  </action>
  <action name="actionAbout_funcatching">
   <property name="text">
    <string>About funcatching</string>
   </property>
  </action>
  <action name="action_New">
   <property name="text">
    <string>&amp;New</string>
   </property>
  </action>
  <action name="action_Open">
   <property name="text">
    <string>&amp;Open</string>
   </property>
  </action>
  <action name="action_Save">
   <property name="text">
    <string>&amp;Save</string>
   </property>
  </action>
  <action name="action_Quit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </action>
  <action name="actionTool_Dialog">
   <property name="text">
    <string>Tool &amp;Dialog</string>
   </property>
  </action>
  <action name="action_About_us">
   <property name="text">
    <string>&amp;About us</string>
   </property>
  </action>
  <action name="actionVersion">
   <property name="text">
    <string>Version</string>
   </property>
  </action>
  <action name="action_View">
   <property name="text">
    <string>&amp;View</string>
   </property>
  </action>
  <action name="actionAdd_new_row">
   <property name="text">
    <string>add new &amp;Row</string>
   </property>
  </action>
  <action name="actionAdd_new_Column">
   <property name="text">
    <string>add new &amp;Column</string>
   </property>
  </action>
  <action name="actionWood">
   <property name="text">
    <string>Wood</string>
   </property>
  </action>
  <action name="actionGlass">
   <property name="text">
    <string>Glass</string>
   </property>
  </action>
  <action name="actionClay">
   <property name="text">
    <string>Clay</string>
   </property>
  </action>
  <action name="actionSet_Row_size">
   <property name="text">
    <string>set Row size</string>
   </property>
  </action>
  <action name="actionSet_Column_size">
   <property name="text">
    <string>set Column size</string>
   </property>
  </action>
  <action name="actionGo_to_cell">
   <property name="text">
    <string>Go to cell</string>
   </property>
  </action>
  <action name="action_Bat">
   <property name="text">
    <string>&amp;Bat</string>
   </property>
  </action>
  <action name="actionTop">
   <property name="text">
    <string>Top</string>
   </property>
  </action>
  <action name="actionButton">
   <property name="text">
    <string>Button</string>
   </property>
  </action>
  <action name="actionLeft">
   <property name="text">
    <string>Left</string>
   </property>
  </action>
  <action name="actionRight">
   <property name="text">
    <string>Right</string>
   </property>
  </action>
  <action name="actionTop_left">
   <property name="text">
    <string>Top-left</string>
   </property>
  </action>
  <action name="actionButton_left">
   <property name="text">
    <string>Button-left</string>
   </property>
  </action>
  <action name="actionTop_right">
   <property name="text">
    <string>Top-right</string>
   </property>
  </action>
  <action name="actionButton_right">
   <property name="text">
    <string>Button-right</string>
   </property>
  </action>
  <action name="actionSize">
   <property name="text">
    <string>Size</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
